resources/assets/js/helpers.js

```diff
@@ -149,41 +175,22 @@ $.extend(true, laravelValidation, {
-            return timezones.indexOf(value) != -1;
-        },
-
-
-        laravelRemote: function (element, attribute, value, token) {
+        dependentElement: function(validator, element, name) {
 
-            var $form = $(element).closest('form');
-            var url=$form.attr('action');
-            var method  = $form.attr('method');
-
-            if (method.toLowerCase()=='get') {
-                token = false;
+            var el=validator.findByName(name);
+            if (el[0]==undefined) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/842620831/issues/source?bid=2225045&fileBranchId=2805859#l181)